column.go

```diff
@@ -57,6 +57,8 @@ func (c *colImpl) Len() int {
 		return len(c.msg.Times)
 	case pb.DType_BOOLEAN:
 		return len(c.msg.Bools)
+	case pb.DType_NULL:
+		return -1
```

Comment:
it already returned -1 (out of the switch case)

Reply:
I know. I wanted to make it explicit and not a "panic? 🤷‍♂️" thing.